website/pages/docs/installation/nx-monorepo-remix.mdx

```diff
@@ -0,0 +1,319 @@
+---
+title: Using Nx monorepo and Remix
```

Comment:
Thanks for this PR @lwhiteley.

Does this need to be scoped to Remix? or could this be a general NX setup guide?

Trying to avoid an explosion of NX + <framework> guides.

Reply:
understandable.

unfortunately, even within a monorepo the build behaviour of different frameworks are not the same.

there can be small differences in the exact steps to set this up between frameworks

We could generalize this but it may probably be a less effective guide.

----

if generalized, maybe a sample repo gradually showcasing different setups could be used but  this can also be less informative if you dont know where to look. Would probably have multiple Readme files outlining the steps in detail either way.

Reply:
I see what you mean.

Looking through the [NX Docs for tailwind](https://nx.dev/recipes/react/using-tailwind-css-in-react), I prefer that it is unrelated to a framework.

We should streamline this for Nx + React, and we can improve as we go. Ideally, including a new step to install nx for the specific framework

Reply:
Makes sense but the diff is probably that there is a NX setup script for tailwind. Streamlining may mean creating an Nx workspace plugin to add pandacss to a framework and handling framework specific stuff there.

Not a bad approach but I guess it's a diff beast.

I could also research this as well to see what could be done

Reply:
i found an issue with the setup and HMR.

Whenever i make any changes in the Remix app, the panda CSS styles totally disappear.

im not sure if this is a remix or panda css issue yet.

I can make changes to the index.css file and they are applied as showcased here: https://github.com/chakra-ui/panda/issues/1021

However, all the styles generated by the utility functions disappear for some reason.

ill have to do some reproduction to isolate the issue in a standalone remix app to see what the difference is